### PR TITLE
make sure some phpdoc fixers are run in right order

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: Symfony/CS/Tests/FixerTest.php [hunk 1]
```diff
@@ -211,12 +211,18 @@ function ($name) {
         // prepare bulk tests for phpdoc fixers to test if:
         // * `phpdoc_to_comment` is first
         // * `phpdoc_indent` is second
+        // * `phpdoc_scalar` is third
         // * `phpdoc_params` is last
         $cases[] = array($fixers['phpdoc_to_comment'], $fixers['phpdoc_indent']);
+        $cases[] = array($fixers['phpdoc_indent'], $fixers['phpdoc_scalar']);
+        $cases[] = array($fixers['phpdoc_scalar'], $fixers['phpdoc_params']);
```
Comment (keradus) [inline Symfony/CS/Tests/FixerTest.php]: this case is checked in that loop below now as well

File: Symfony/CS/Tests/FixerTest.php [hunk 2]
```diff
@@ -211,12 +211,18 @@ function ($name) {
...
+
+        $cases[] = array($fixers['phpdoc_to_comment'], $fixers['phpdoc_indent'], $fixers['phpdoc_scalar']);
```
Comment (keradus) [inline Symfony/CS/Tests/FixerTest.php]: 3 elements in array? won't the 3rd one will be ignored ?

Comment (GrahamCampbell) [inline Symfony/CS/Tests/FixerTest.php]: Yes

**Comment by SpacePossum** [comment]
thanks @keradus and @GrahamCampbell fore reviewing, PR updated.

**Comment by keradus** [comment]
:+1:

**Comment by gharlan** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
Sorry I messed this up in the first place. :)

**Comment by SpacePossum** [comment]
>Sorry I messed this up in the first place. :)
all good, thanks for making the fixers in the first place :)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**